code/game/gamemodes/shadowling/shadowling.dm

```diff
@@ -300,7 +300,7 @@ Made by Xhuis
 	blood_color = "#555555"
 	flesh_color = "#222222"
 
```

Comment:
Nah, since all Shadowling types are children of the shadow people, themselves, which only have brains, as far as I can tell.